projects/libwpd/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+pushd librevenge
+./autogen.sh
+./configure --without-docs --disable-shared --enable-static --disable-tests
+make -j$(nproc)
+rvnginc=$(pwd)/inc
+rvnglib=$(pwd)/src/lib
+popd
+
+./autogen.sh
```

Comment:
It is not needed--I guess I should remove the line from librevenge (and from everywhere else I used it).